app/src/main/java/co/omise/android/example/PaymentSettingFragment.kt

```diff
@@ -39,14 +39,13 @@ class PaymentSettingFragment : PreferenceFragmentCompat() {
                 .findPreference<CheckBoxPreference>(getString(R.string.payment_preference_is_use_specifics_payment_methods_key))?.isChecked
                 ?: false
 
-        PaymentSetting.getPaymentMethodSettingKeysWithSourceTypes(context!!).forEach {
-            preferenceScreen.findPreference<CheckBoxPreference>(it.first)?.apply {
+        PaymentSetting.getPaymentMethodPreferences(context!!).forEach {
```

Comment:
There is no way to get rid of the `!!` over here is there? 🤔

Reply:
Yeah, I think so.